web/src/app/administration/components/administration-overview/administration-overview.component.ts

```diff
@@ -35,8 +35,9 @@ export class AdministrationOverviewComponent implements OnInit {
   }
 
   ngOnInit() {
-    this.routingAccess$ = this.taskanaEngineService.isCustomRoutingRulesEnabled$;
-    this.routingAccess$.pipe(takeUntil(this.destroy$)).subscribe();
+    this.taskanaEngineService.isCustomRoutingRulesEnabled().subscribe((value) => {
```

Comment:
we need to destroy the subscription when component is destroyed with `this.taskanaEngineService.isCustomRoutingRulesEnabled().pipe(takeUntil(this.destroy$)).subscribe....`